website/src/documentation/programming-guide.md

```diff
@@ -2253,13 +2253,13 @@ into fixed windows, each one minute in length:
 #### 7.3.2. Sliding time windows {#using-sliding-time-windows}
 
 The following example code shows how to apply `Window` to divide a `PCollection`
-into sliding time windows. Each window is 30 minutes in length, and a new window
+into sliding time windows. Each window is 30 seconds in length, and a new window
```

Comment:
I don't think we fully migrate the website into the beam code base yet. We might still need to update this in the beam-site. @alanmyrvold correct me if I am wrong.

Reply:
Yes, the website migration is not finished so website changes need to be made in the beam-site repo's src directory for now. https://github.com/apache/beam-site

Reply:
Done! apache/beam-site#559